src/util/types.ts

```diff
@@ -1235,7 +1235,7 @@ type TooltipBoxLayoutOption = Pick<
 /**
  * Position relative to the hoverred element. Only available when trigger is item.
  */
-interface PositionCallback {
+export interface PositionCallback {
```

Comment:
PositionCallback can be renamed to TooltipPositionCallback to avoid potential conflicts when exported